src/Prometheus.Client.MetricServer/MetricServer.HttpListener.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if NET45
+#if NET47
```

Comment:
Most likely it should be netstandard2.0 directive instead, as HttpListerner is available in netstandard 2.0

Reply:
changed to NETSTANDARD20